src/server/set_family.cc

```diff
@@ -36,6 +36,48 @@ using SetType = pair<void*, unsigned>;
 
 namespace {
 
+struct ScanOpts {
```

Comment:
@romange does it makes sense to share this struct in common.h so that sscan and hscan will use it?